hack/community/generate.sh

```diff
@@ -0,0 +1,140 @@
+    '.spec.install.spec.deployments[0].spec.template.spec.containers[0].image |= strenv(QUAY_IMAGE)' \
+    "${CO_CSV}"
+
+    echo "update maturity"
+    /tmp/yq eval --exit-status --inplace \
+    '.spec.maturity |= strenv(maturity)' "${CO_CSV}"
+
+    echo "update operator version"
+    /tmp/yq eval --exit-status --inplace \
+    '.spec.version |= strenv(OPERATOR_VERSION)' "${CO_CSV}"
+}
+
+# TODO ensure minor versions accounted for
+generate_manifests() {
+  if test -e "$FILE"; then
+      IFS='.' read -r -a version_split <<< "${OPERATOR_VERSION}"
+      OPERATOR_VERSION="${version_split[0]}.${version_split[1]}.$((1 + 10#${version_split[2]}))"
+      echo "Cutting new community release version $OPERATOR_VERSION"
+  else
+      echo "  Cutting community release with version $OPERATOR_VERSION"
+  fi
+
+  echo
+  echo "Update operator manifests"
+  cp -r "${OPERATOR_MANIFESTS}" "${CO_OPERATOR_DIR}/${OPERATOR_VERSION}"
+  pushd "${CO_DIR}"
+  git checkout -B "${OPERATOR_VERSION}"
+  CO_CSV="${CO_OPERATOR_DIR}/${OPERATOR_VERSION}/${CSV}"
+
+  replace
+}
+
+submit_PR() {
+  echo
+  echo "Submit PR to community operators"
+  echo "commit changes"
+  git add --all
+  git commit -s -m "community-windows-machine-config-operator.v${OPERATOR_VERSION}"
+  git push --set-upstream --force origin HEAD
+
+  echo "submit PR"
+  # Create PR
+  curl "https://api.github.com/repos/${CO_REPO}/pulls" --user "${GITHUB_USER}:${GITHUB_TOKEN}" -X POST \
+     --data '{"title": "'"$(git log -1 --format=%s)"'", "base": "main", "body": "An automated PR to update
+      community-windows-machine-config-operator to v'"${OPERATOR_VERSION}"'", "head":
+      "'"${GITHUB_USER}:${OPERATOR_VERSION}"'"}'
+  popd
+
+  echo "Done"
+}
+
+# Install tools:yq
+if ! command -v which yq &> /dev/null; then
+  curl -L https://github.com/mikefarah/yq/releases/download/v4.13.5/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+  else
+    echo "yq already installed"
+fi
+
+OPERATOR_MANIFESTS="bundle/manifests"
+CSV="${OPERATOR_MANIFESTS}/windows-machine-config-operator.clusterserviceversion.yaml"
+
+echo "Extracting WMCO version from CSV"
+OPERATOR_VERSION=$(/tmp/yq eval '.spec.version' "${CSV}")
+echo "Current WMCO version: ${OPERATOR_VERSION}"
+
+FILE=../community-operators-prod/operators/community-windows-machine-config-operator/${OPERATOR_VERSION}
+
+CO_PROJECT="community-operators-prod"
+CO_REPO="redhat-openshift-ecosystem/${CO_PROJECT}"
+CO_GIT="https://github.com/${CO_REPO}.git"
+CO_FORK="https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com/${GITHUB_USER}/${CO_PROJECT}.git"
+CO_DIR=$(mktemp -d)
+CO_OPERATOR_DIR="${CO_DIR}/operators/community-windows-machine-config-operator"
+
+if [ -z "${GITHUB_USER}" ] || [ -z "${GITHUB_TOKEN}" ] || [ -z "${GITHUB_NAME}" ] || [ -z "${GITHUB_EMAIL}" ] || [ -z \
+"${QUAY_IMAGE}"]; then
+    echo "Must set all of GITHUB_USER, GITHUB_TOKEN, GITHUB_NAME, GITHUB_EMAIL, and QUAY_IMAGE"
+    exit 1
+fi
+
+clone_community
+generate_manifests
+submit_PR
```

Comment:
Consider introducing an optional arg to the script where you can run it without a PR being created, if that'd be useful. Might also be useful helpful for your local dev

So usage would be:
```bash
generate.sh [--no-pr]
```

Reply:
That could be a useful option. What does @openshift/openshift-team-windows-containers think?

Reply:
I think that would be useful